### PR TITLE
Simplify column computation and add types to Ecto results

### PR DESCRIPTION
**Pull request by jonatanklosko**
image

### CODE REVIEW

File: lib/kino/utils/table.ex [hunk 1]
```diff
@@ -1,77 +1,75 @@
...
   end
 
+  @doc """
+  Computes table column specifications for the given Ecto schema.
+  """
+  @spec columns_for_schema(module()) :: list(Kino.Table.column())
+  def columns_for_schema(schema) do
+    for field <- schema.__schema__(:fields) do
+      type = schema.__schema__(:type, field)
+      %{key: field, label: inspect(field), type: to_string(type)}
```
Comment (josevalim) [inline lib/kino/utils/table.ex:65]: Composite and parameterized types can be tuples. :)

File: lib/kino/utils/table.ex [hunk 2]
```diff
@@ -1,77 +1,75 @@
...
-        else
-          all_keys
-        end
+      nil -> []
+      first_record -> columns_for_record(first_record)
     end
   end
 
-  defp keys_for_record(record) when is_tuple(record) do
+  defp columns_for_record(record) when is_tuple(record) do
```
Comment (josevalim) [inline lib/kino/utils/table.ex:24]: This clause can be removed?

**Review by josevalim — COMMENTED** [review]
Just one tiny comment about Ecto and ship it!

File: lib/kino/utils/table.ex (hunk 1 above)
Comment (jonatanklosko) [reply lib/kino/utils/table.ex:65]: Good point, so in that case `inspect` should look fine, right?

image

**Pull request merged**